pycript/Reqcheck.py

```diff
@@ -189,55 +189,63 @@ def decrypt_and_update_parameter_keys_and_values(extender, currentreq, decryptio
 def encrypt_and_update_parameters(extender, currentreq, encryptionpath, selected_method, selectedlang, body, parameters, header,selected_request_inc_ex_ctype,listofparam):
     for param in parameters:
         if selected_method == "GET" and param.getType() == IParameter.PARAM_URL:
-            encrypted_param_value = Parameterencrypt(selectedlang, encryptionpath, param.getValue())
+            parameter_value = param
+            encrypted_param_value = update_raw_value(parameter_value, selectedlang, encryptionpath, Parameterencrypt,selected_request_inc_ex_ctype, listofparam)
             currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), encrypted_param_value, param.getType()))
-
         elif selected_method == "BODY" and param.getType() != IParameter.PARAM_URL:
             if param.getType() == IParameter.PARAM_BODY:
-                encrypted_param_value = Parameterencrypt(selectedlang, encryptionpath, param.getValue())
+                parameter_value = param
+                encrypted_param_value = update_raw_value(parameter_value, selectedlang, encryptionpath, Parameterencrypt,selected_request_inc_ex_ctype, listofparam)
                 currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), encrypted_param_value, param.getType()))
+                
             elif param.getType() == IParameter.PARAM_JSON:
                 json_object = loads(body)
                 json_object = update_json_value(json_object, selectedlang, encryptionpath, Parameterencrypt,selected_request_inc_ex_ctype,listofparam)
                 output = extender.helpers.stringToBytes(dumps(json_object,separators=(',', ':')))
                 currentreq = extender.helpers.buildHttpMessage(header, output)
                 break
         
+            else:
```

Comment:
` if param.getType() == IParameter.PARAM_URL:` added inside the `elif selected_method == "BODY" and param.getType() != IParameter.PARAM_URL:`  


 `elif selected_method == "BODY" and param.getType() != IParameter.PARAM_URL:`   will always exclude the get parameter, means ` if param.getType() == IParameter.PARAM_URL:` is never going to trigged and get parameter will be excluded 
 
 ` if param.getType() == IParameter.PARAM_URL:` should not be part of the another nested condition